SECURITY.md

```diff
@@ -0,0 +1,174 @@
+# Security Policy
```

Comment:
Maybe we should eventually roll this up into some kind of overall slsa-framework or OpenSSF-wide security policy/reporting process?

@MarkLodato @laurentsimon @asraa @joshuagl Thoughts?

Reply:
At the very least a lot of this should be shared between slsa-github-generator and slsa-verifier.

Reply:
org-level makes sense to me too! We could file an issue on https://github.com/slsa-framework/governance to track

Reply:
Added issue https://github.com/slsa-framework/governance/issues/19 to track.